style.css

```diff
@@ -3,6 +3,34 @@
 @media only screen and (min-width: 375px) and (max-width: 768px) {
   /* My body config */
 
+  .nav-desktop {
```

Comment:
[Optional] Please consider moving all your images into a single folder. This will improve your directory structure and declutter your root folder